api/institutions/views.py

```diff
@@ -581,6 +582,53 @@ def get_default_search(self):
         )
 
 
+class _NewInstitutionSummaryMetricsList(JSONAPIBaseView, generics.RetrieveAPIView, InstitutionMixin):
+    '''list view for institution-summary metrics
+
+    used only when the INSTITUTIONAL_DASHBOARD_2024 feature flag is active
+    (and should be renamed without "New" when that flag is permanently active)
+    '''
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+        IsInstitutionalMetricsUser,
+    )
+
+    required_read_scopes = [CoreScopes.INSTITUTION_METRICS_READ]
+    required_write_scopes = [CoreScopes.NULL]
+
+    view_category = 'institutions'
+    view_name = 'institution-summary-metrics'
+
+    serializer_class = NewInstitutionSummaryMetricsSerializer
+
+    def get_object(self):
+        institution = self.get_institution()
+        search_object = self.get_default_search()
+        if search_object:
+            object = search_object.execute()[0]
+            object.id = institution._id
+            return object
+
+    def get_default_search(self):
+        _yearmonth = InstitutionMonthlySummaryReport.most_recent_yearmonth()
+        if _yearmonth is None:
+            return None
+        return (
+            InstitutionMonthlySummaryReport.search()
+            .filter('term', report_yearmonth=str(_yearmonth))
+            .filter('term', institution_id=self.get_institution()._id)
+        )
+
+
+institution_summary_metrics_list_view = toggle_view_by_flag(
```

Comment:
if you make it a list view, this is fine, but if it's a detail view (as implemented now) it shouldn't be called `list_view`